movies/movies/FilmViewController.swift

```diff
@@ -0,0 +1,289 @@
+
+            starImageView.topAnchor.constraint(equalTo: genresLabel.bottomAnchor, constant: 15),
+            starImageView.leadingAnchor.constraint(equalTo: boxView.leadingAnchor, constant: 20),
+            starImageView.widthAnchor.constraint(equalToConstant: 20),
+            starImageView.heightAnchor.constraint(equalToConstant: 20),
+
+            rateLabel.bottomAnchor.constraint(equalTo: starImageView.bottomAnchor, constant: 2),
+            rateLabel.leadingAnchor.constraint(equalTo: starImageView.trailingAnchor, constant: 10),
+
+            taglineLabel.topAnchor.constraint(equalTo: starImageView.bottomAnchor, constant: 15),
+            taglineLabel.leadingAnchor.constraint(equalTo: boxView.leadingAnchor, constant: 20),
+            taglineLabel.widthAnchor.constraint(equalTo: boxView.widthAnchor, constant: -40),
+
+            descriptionTitleLabel.topAnchor.constraint(equalTo: taglineLabel.bottomAnchor, constant: 8),
+            descriptionTitleLabel.leadingAnchor.constraint(equalTo: boxView.leadingAnchor, constant: 20),
+
+            descriptionLabel.topAnchor.constraint(equalTo: descriptionTitleLabel.bottomAnchor, constant: 8),
+            descriptionLabel.leadingAnchor.constraint(equalTo: boxView.leadingAnchor, constant: 20),
+            descriptionLabel.trailingAnchor.constraint(equalTo: boxView.trailingAnchor, constant: -20),
+            descriptionLabel.bottomAnchor.constraint(equalTo: boxView.bottomAnchor, constant: 20),
+
+            webViewButton.leadingAnchor.constraint(equalTo: filmImageView.leadingAnchor, constant: 10),
+            webViewButton.topAnchor.constraint(equalTo: filmImageView.topAnchor, constant: 250),
+            webViewButton.heightAnchor.constraint(equalToConstant: 40),
+            webViewButton.widthAnchor.constraint(equalToConstant: 40),
+
+            rightImageView.heightAnchor.constraint(equalToConstant: 20),
+            rightImageView.widthAnchor.constraint(equalToConstant: 20),
+            rightImageView.topAnchor.constraint(equalTo: webViewButton.topAnchor, constant: 10),
+            rightImageView.leadingAnchor.constraint(equalTo: webViewButton.leadingAnchor, constant: 12),
+
+            trailerLabel.leadingAnchor.constraint(equalTo: webViewButton.trailingAnchor, constant: 10),
+            trailerLabel.bottomAnchor.constraint(equalTo: webViewButton.bottomAnchor, constant: 2)
+
+        ])
+    }
+
+    private func setupData(data: Film) {
+        filmImageView.loadImage(with: data.poster)
+        titleLabel.attributedText = NSMutableAttributedString().normal("\(data.title) ")
+            .normalGray("(\(data.release.prefix(4)))")
+        rateLabel.text = "\(data.rate)" + Constants.imdbFullRate
+        taglineLabel.text = "\(data.tagline)"
+        descriptionLabel.text = data.overview
+        genresLabel.text = data.genres.map(\.name)
+            .joined(separator: ", ") + Constants.dot + "\((data.runtime) / 60) ч \((data.runtime) % 60) мин"
```

Comment:
```suggestion
            .joined(separator: ", ") "
```
 текстовки убираем и вместо конкатенации используем интерполяцию

Reply:
литер ругается на длинную строку, приходится один раз разделить конкантенацией